api/src/test/java/org/openmrs/validator/ConceptValidatorTest.java

```diff
@@ -35,46 +35,53 @@
  * Tests methods on the {@link ConceptValidator} class.
  */
 public class ConceptValidatorTest extends BaseContextSensitiveTest {
-	
+
 	/**
 	 * @verifies {@link ConceptValidator#validate(Object,Errors)} test = should fail if there is a
 	 *           duplicate unretired concept name in the locale
 	 */
-	@Test(expected = DuplicateConceptNameException.class)
+
+	@Test
 	@Verifies(value = "should fail if there is a duplicate unretired concept name in the locale", method = "validate(Concept)")
 	public void validate_shouldFailIfThereIsADuplicateUnretiredConceptNameInTheLocale() throws Exception {
 		Context.setLocale(new Locale("en", "GB"));
 		Concept concept = Context.getConceptService().getConcept(5497);
 		String duplicateName = concept.getFullySpecifiedName(Context.getLocale()).getName();
-		
+
 		ConceptName newName = new ConceptName(duplicateName, Context.getLocale());
 		newName.setDateCreated(Calendar.getInstance().getTime());
 		newName.setCreator(Context.getAuthenticatedUser());
 		concept.addName(newName);
 		Errors errors = new BindException(concept, "concept");
-		new ConceptValidator().validate(concept, errors);
+
+		try {
```

Comment:
The reason I used try-catch instead of ExpectedException is because for some reason, ExpectedException cannot catch exact error message. If the actual error message is a subset of the expected message, it validates it to be true. I tried it before to make it cleaner code, but since it did not work for our purpose, I changed it to try-catch